data/moves.js

```diff
@@ -284,6 +284,7 @@ let BattleMovedex = {
 			let action = this.willMove(target);
 			if (action) {
 				this.cancelMove(target);
+				action.priority = 7.1;
 				this.queue.unshift(action);
```

Comment:
Should this just use `prioritizeAction`?

Reply:
Probably, I'll test it.